### ISSUE TITLE
bug: if image no alt text it cannot parser

### ISSUE THREAD

**Issue by Paul-16098** [johnlepikhin/markdown-ppp#7]
```md
![](image.png)
![has alt](image.png)
```
```html
<p>![](image.png)</p>
<p><img src="image.jpg" alt="has alt"></p>
```

**Comment by johnlepikhin** [comment]
Hi,

Fixed in v2.1.1.
